pom.xml

```diff
@@ -254,10 +253,27 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <!-- inheriting general ossrh-release setup from aissemble-parent -->
+            <id>ossrh-release</id>
+            <properties>
+                <maven.build.cache.enabled>false</maven.build.cache.enabled>
+            </properties>
+        </profile>
     </profiles>
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.sonatype.plugins</groupId>
+                    <artifactId>nexus-staging-maven-plugin</artifactId>
+                    <configuration>
+                        <!-- how long to wait for the staging repository to be closed and released -->
+                        <stagingProgressTimeoutMinutes>30</stagingProgressTimeoutMinutes>
```

Comment:
I: Default is 5 mins which turns out to be just shy of how long it takes all the aissemble maven artifacts to deploy from staging to release.  Based an RC release and the official release, this seems to take between 5-7 minutes, so 30 minutes may be overkill, but I wanted to collect more information about how long it can take before bringing the threshold down.